wiki/en/Running-a-Server.md

```diff
@@ -193,37 +186,13 @@ Now you can run these with the `systemctl` command, for example:
 
 You can see the result of these commands if you run `systemctl status jamulus`, or by viewing the logs.
 
-### Making a Server status page
-
-With the `-m` command line argument, Server statistics can be generated to be put on a web page.
-
-Here is an example php script using the Server status file to display the current Server status on a html page (assuming the following command line argument to be used: `-m /var/www/stat1.dat`):
-
-~~~
-<?php
-function loadserverstat ( $statfilename )
-{
-   $datei = fopen ( $statfilename, "r" );
-   while ( !feof ( $datei ) )
-   {
-          $buffer = fgets ( $datei, 4096 );
-          echo $buffer;
-   }
-   fclose($datei);
-}
-?>
-<?php loadserverstat ( "stat1.dat" ); ?>
-~~~
-
 
 # Servers on the desktop
 
-Jamulus can be configured to run in Server mode on the desktop. This gives you a graphical user interface to control most of the settings. 
+Jamulus can be configured to run in Server mode on the desktop. This gives you a graphical user interface to control most of the settings.
```

Comment:
> Jamulus can be configured to run in Server mode on the desktop.

Should read

"Jamulus can be run in Server mode from the desktop."

Reply:
Changed.